### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by codenirvana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+# newman-reporter-html
+Advanced HTML reporter for [newman](https://github.com/postmanlabs/newman).
+
+## Install
+```terminal
+$ npm install -g newman-reporter-html
+```
+
+## Usage
+In order to use this reporter, ensure that you have installed this at the same level as of newman.
```
Comment (kunagpal) [inline README.md]: Clarify

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+| `--reporter-html-template <path>` | Specify a path to the custom template which will be used to render the HTML report. This option depends on `--reporter html` and `--reporter-html-export` being present in the run command. If this option is not specified, the [default template](https://github.com/postmanlabs/newman-reporter-html/blob/develop/lib/template-default.hbs) is used |
+
+Custom templates (currently handlebars only) can be passed to the HTML reporter via `--reporter-html-template <path>` with `--reporters html` and `--reporter-html-export`.
+The [default template](https://github.com/postmanlabs/newman-reporter-html/blob/develop/lib/template-default.hbs) is used in all other cases.
+
+## Compatibility
+
+| **newman-reporter-html** | **newman** | **node** |
+|:------------------------:|:----------:|:--------:|
+|          v0.1.0          | >= v3.9.5  | >= v6.x  |
```
Comment (kunagpal) [inline README.md]: `>=3.9.4`

Comment (kunagpal) [reply README.md]: Yes, but the next release will not be 3.9.5

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
-# newman-reporter-html
\ No newline at end of file
+# newman-reporter-html
+Advanced HTML reporter for [newman](https://github.com/postmanlabs/newman).
```
Comment (shamasis) [inline README.md]: Just HTML reporter and not Advanced

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+
+## Install
```
Comment (shamasis) [inline README.md]: Need a paragraph before this to explain that this is a reporter for Newman that provides HTML output and that it needs to be used in conjunction. And that when installed, Newman CLI will recognise the following options.

File: README.md [hunk 5]
```diff
@@ -1 +1,45 @@
...
+### Options
+
+| CLI Option  | Description       |
+|-------------|-------------------|
+| `--reporter-html-export <path>` | Specify a path where the output HTML file will be written to disk. If not specified, the file will be written to `newman/` in the current working directory. |
+| `--reporter-html-template <path>` | Specify a path to the custom template which will be used to render the HTML report. This option depends on `--reporter html` and `--reporter-html-export` being present in the run command. If this option is not specified, the [default template](https://github.com/postmanlabs/newman-reporter-html/blob/develop/lib/template-default.hbs) is used |
+
+Custom templates (currently handlebars only) can be passed to the HTML reporter via `--reporter-html-template <path>` with `--reporters html` and `--reporter-html-export`.
+The [default template](https://github.com/postmanlabs/newman-reporter-html/blob/develop/lib/template-default.hbs) is used in all other cases.
+
```
Comment (shamasis) [inline README.md]: Need to mention the programmatic options Newman will start accepting once this reporter is installed. 

There has to be a troubleshooting section
- when reporter is not found

**Pull request merged**